### DISCUSSION TITLE
Can't search any network flow in the "netobserv-privileged" namespace

### DISCUSSION THREAD

**Discussion by xela-engineer** [netobserv/network-observability-operator#337] [Q&A] [answered]
I tried to search network flow in the "netobserv-privileged" namespace.
However, the network graph console show nothing.
And I also observe that there is no label "openshift.io/cluster-monitoring" on the "netobserv-privileged" namespace.
Is it a normal behavior? and what is the reason? @_@

**Comment by jotak** [comment, marked as answer]
This has already been closed, but let me try to answer anyway:
- there are indeed pods deployed by netobserv in the "netobserv-privileged" namespace, which communicate with pods deployed in "netobserv" namespace. But, unless netobserv is configured to use Kafka, this traffic uses the host network, hence the resulting flows have IPs that correspond to the nodes and not to the pods, which makes it impossible to correlate them with a pod or a namespace. Any pods using host-network will be in the same situation.
- on "openshift.io/cluster-monitoring" label, currently our eBPF agent pods don't expose metrics so there's no need for such a label. This could change in the future.

**Reply by xela-engineer** [reply]
Thank you so much!
